### ISSUE TITLE
training problem

### ISSUE THREAD

**Issue by chenshen03** [deepinx/iqiyi-vid-challenge#2]
What results can you achieve in the IQIYI_VID competition with this code and the pre-training model?

In addition, I first use your code to extract the facial features of the IQIYI_VID data set, which are then used to train the MLP network. During training, **the acc is 0 and the loss is NAN**. What is the reason for this? Thank you.

![image](https://user-images.githubusercontent.com/20432947/54919699-791bae00-4f3c-11e9-9db3-6d5368b609fa.png)

**Comment by chenshen03** [comment]
The problem has been solved.
